assets/js/pages/Users/UsersPage.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { listUsers, deleteUser } from '@lib/api/users';
+
+import { toast } from 'react-hot-toast';
+
+import Users from './Users';
+
+export const SUCCESS_DELETE_MESSAGE = 'User deleted successfully';
+export const ERROR_DELETE_MESSAGE = 'An error occurred: User not found';
```

Comment:
Better `USER_NOT_FOUND_ERROR` maybe
By the way, I wouldn't export them, just for test convenience, you can just write the same text in the tests